Dfe.Academies.External.Web/CypressTests/cypress/page-objects/pages/A2BFAMTrustOpeningDateDetails.ts

```diff
@@ -0,0 +1,32 @@
+import BasePage from "../BasePage"
+export default class A2BFAMTrustOpeningDateDetails extends BasePage {
+    static selectDayAndInput()
+    {
+        cy.get('#sip_formtrustopeningdate-day').click()
+        cy.get('#sip_formtrustopeningdate-day').type('1')
+    }
+
+    static selectMonthAndInput()
+    {
+        cy.get('#sip_formtrustopeningdate-month').click()
+        cy.get('#sip_formtrustopeningdate-month').type('9')
+    }
+
+    // BELOW FUNCTION USES LOGIC FROM FIXTURES AND COMMANDS.TS
+    static selectYearAndInput()
+    {   
+        cy.selectYearAndInput()
+    }
+
+    static inputApproverDetailsAndSubmit()
+    {
+        cy.get('#TrustApproverName').click()
+        cy.get('#TrustApproverName').type('James Stewart')
+
+        cy.get('#TrustApproverEmail').click()
+        cy.get('#TrustApproverEmail').type('james.stewart@aol.com')
```

Comment:
Is this a real email? If so probably best using one from a library like [Faker](https://fakerjs.dev/api/internet.html#email)

Reply:
We just need to be careful in case that's a real email is all. Don't want some random getting emails from our systems :D

Faker should mean it's not an issue.